extensions/blocks/podcast-player/style.scss

```diff
@@ -62,6 +60,13 @@ $player-float-background: $light-gray-200;
 	.jetpack-podcast-player {
 		padding-top: 0;
 		padding-bottom: 0;
+
+		/**
+		* Set default values for our CSS variables.
+		*/
+		--jetpack-podcast-player-primary: #{$jetpack-podcast-player-primary};
+		--jetpack-podcast-player-secondary: #{$jetpack-podcast-player-secondary};
+		--jetpack-podcast-player-background: #{$jetpack-podcast-player-background};
```

Comment:
Is this necessary when we are also defaulting all values in all their respective declarations like this? 

```
color: $jetpack-podcast-player-secondary;
color: var( --jetpack-podcast-player-secondary );
```

Reply:
Yes, this is necessary as this sets the default value for those variables. `color: var( --jetpack-podcast-player-secondary );` is valid even when that variable has no value assigned. So `color: $jetpack-podcast-player-secondary;` would not apply in browsers supporting CSS variables. To combat this, one can usually pass a fallback value into the var() declaration like so `color: var( --jetpack-podcast-player-secondary, $jetpack-podcast-player-secondary );`. This doesn't work with the postcss postprocessing applied in Jetpack though. This is why we're setting the default values in this block. On the upside, defining them once does look a bit cleaner in code rather than passing a fallback in each var declaration.